src/routes/safe/components/Transactions/TxList/TxDataRow.tsx

```diff
@@ -21,29 +22,42 @@ type TxDataRowType = {
   hasExplorer?: boolean
   title: string
   value?: string
+  isArray?: boolean
+  method?: string
+  paramType?: string
 }
 
-export const TxDataRow = ({ children, inlineType, hasExplorer = true, title, value }: TxDataRowType): ReactElement => (
+export const TxDataRow = ({
+  children,
+  inlineType,
+  hasExplorer = true,
+  title,
+  value,
+  isArray,
+  method,
+  paramType,
+}: TxDataRowType): ReactElement => (
   <StyledGridRow>
-    <Text size="xl" as="span">
+    <Text size="xl" as="span" color="placeHolder">
       {title}
     </Text>
-    {value && inlineType === 'hash' && (
-      <InlineEthHashInfo
+    {isArray && value && method && paramType && <Value method={method} type={paramType} value={value as string} />}
+    {value && typeof value === 'string' && inlineType === 'hash' && (
```

Comment:
Need you check the `typeof` here and on line 54?

Reply:
No. Again, leftover. I changed the `value` type by last and didn't clean this here.